packages/expo-font/src/ExpoFontLoader.web.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import WebFont from 'webfontloader';
+
+export default {
+  get name(): string {
+    return 'ExpoFontLoader';
+  },
+  loadAsync(fontFamilyName: string, fontUri: string): Promise {
+    return new Promise((resolve, reject) => {
+      WebFont.load({
+        active: resolve,
+        inactive() {
+          reject(
+            new Error('ExpoFontLoader.loadAsync(): The browser does not support linked fonts')
```

Comment:
The WebFontLoader docs say "This event is triggered when the browser does not support linked fonts or if none of the fonts could be loaded." -- include the that last bit too since it's probably more common (bad internet connection, for example).